### PR TITLE
Add changes for OpenJ9 0.14.2 release

### PR DESCRIPTION
**Pull request by SueChaplain**
- Add OpenSSL 1.0.1 support
- Digest algorithm disabled by default

CLoses: #269
Closes: #260

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
@pshipton @groeges - please review

File: docs/version0.14.2.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
+
+### Support for OpenSSL 1.0.1
+
+OpenSSL version 1.0.1 support is now enabled; Earlier releases supported only OpenJDK 1.0.2 and 1.1.x. On Linux&reg; and AIX&reg; platforms, the OpenSSL libraries are expected to be available on the system path. For more information about cryptographic acceleration with OpenSSL, see [Cryptographic operations](introduction.md#cryptographic-operations).
+
+### OpenSSL Digest algorithm disabled
+
+Due to issue [#5611](https://github.com/eclipse/openj9/issues/5611), the Digest algorithm is disabled. As an alternative, the CBC, GCM, and RSA cryptographic algorithms can be used for cryptographic acceleration.
```
Comment (pshipton) [inline docs/version0.14.2.md]: CBC, GCM, and RSA are not alternatives to MessageDigest.

Comment (SueChaplain) [reply docs/version0.14.2.md]: Updated

File: docs/djdknativedigest.md [hunk 2]
```diff
@@ -26,6 +26,8 @@
 
 This option enables or disables OpenSSL native cryptographic support for the Digest algorithm.
 
+<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Restriction:** ![Start of content that applies to Java 8 (LTS)](cr/java8.png) ![Start of content that applies to Java 11 (LTS)](cr/java11.png) Due to issue [#5611](https://github.com/eclipse/openj9/issues/5611), the Digest algorithm is currently disabled. ![End of content that applies only to Java 8 and 11 (LTS)](cr/java_close_lts.png)
+
```
Comment (pshipton) [inline docs/djdknativedigest.md]: Should the default in the chart be changed? It is somewhat misleading.

Comment (SueChaplain) [reply docs/djdknativedigest.md]: I was assuming that the default applied to JDK 12? Is that not true?

Comment (pshipton) [reply docs/djdknativedigest.md]: It is true the default applies to JDK 12, but it's not clear the table only applies to JDK 12. Also we expect very few people to be reading the docs for JDK 12 as the primary users are JDK 8 and 11.

**Comment by SueChaplain** [comment]
Jenkins doc stage

File: docs/djdknativedigest.md (hunk 2 above)
Comment (SueChaplain) [reply docs/djdknativedigest.md]: I added an icon in the table. Does that help?
See staging: https://eclipse.github.io/openj9-docs-staging/271/djdknativedigest/

Comment (pshipton) [reply docs/djdknativedigest.md]: Yes. Should `false` also have a checkmark and a tag for jdk 8 and 11?

Comment (SueChaplain) [reply docs/djdknativedigest.md]: I assume for 8 & 11 the option has no effect. So would it be misleading to suggest that there is a default?

Comment (pshipton) [reply docs/djdknativedigest.md]: I guess. There is a setting, which is false, and it cannot be modified.

Comment (pshipton) [reply docs/djdknativedigest.md]: Let me know if you are done, and I will merge.

**Pull request merged**